decidim-meetings/app/controllers/decidim/meetings/meetings_controller.rb

```diff
@@ -6,9 +6,33 @@ module Meetings
     class MeetingsController < Decidim::Meetings::ApplicationController
       include FilterResource
 
-      helper_method :meetings, :geocoded_meetings, :meeting
+      helper_method :meeting
 
-      def index; end
+      def index
+        results = search.results
+
+        if !params[:filter] && results.length == 0
+          params[:filter] = {
+            date: "past",
+            meetings_warning: true
+          }
+          results = search_klass.new(search_params).results
+          if results.length == 0
+            params[:filter] = {
```

Comment:
I think we need to take another approach on this, and create a different `search` object instead of creating the `results` object, this way we can still use all the helper methods.